{"name": "Тест: при неправильном передаче номера заказа, запрос вернет ощибку", "status": "passed", "description": "Генерирую даные курьера, получаю его id. Принимаю заказ курьером, ввожу id курьера и несуществующий track заказа в query-параметры, проверяю код 404, и тело ответа", "attachments": [{"name": "log", "source": "2be8d3e6-55dc-4c7e-856b-720a69570b78-attachment.txt", "type": "text/plain"}], "start": 1717768228686, "stop": 1717768229542, "uuid": "4c87c670-d429-428b-81d8-acf3e8fe67a0", "historyId": "5e38a94ab6d36ad9837c9b631374b92a", "testCaseId": "5e38a94ab6d36ad9837c9b631374b92a", "fullName": "tests.test_assept_order.TestAsseptOrder#test_assept_order_with_incorrect_query_parameter_id_order_shows_code_400_conflict", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_assept_order"}, {"name": "subSuite", "value": "TestAsseptOrder"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "8512-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_assept_order"}]}